wom/src/main/scala/wom/types/WomFloatType.scala

```diff
@@ -9,6 +9,7 @@ case object WomFloatType extends WomPrimitiveType {
   val toDisplayString: String = "Float"
 
   override protected def coercion = {
+    case i: Int => WomFloat(i.toDouble)
```

Comment:
This [is not valid under WDL](http://virtualenvwrapper.readthedocs.io/en/latest/), though I'm not 100% sure how much of WOM leaks into WDL/CWL.

ToL: I'm starting to think coercion implementations may not belong in WOM? I'm running into similar questions around complex CWL files/directories.

Reply:
Yeah I thought I needed this but I actually don't I'll revert it